app/src/main/admin/components/MainMenu/MainMenuNew.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from "react";
```

Comment:
This file will have to be renamed to MainMenu.tsx and the old one deleted in next PR, this is temporary file to be able to generate breadcrumbs for redux pages.